tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/TestHelpers.cs

```diff
@@ -150,8 +150,8 @@ public static string DescribeTestFolder(Assets assets, string[] sampleFiles, str
             if (isPushTest)
             {
                 string adjustedAssetsRepoBranch = string.Format("test_{0}_{1}", testGuid, assets.TagPrefix);
-                // Call InitIntegrationBranch
-                InitIntegrationBranch(assets, adjustedAssetsRepoBranch);
+                // Call InitIntegrationTag
+                InitIntegrationTag(assets, adjustedAssetsRepoBranch);
```

Comment:
I do like that you've changed the function names to change branch to tags, along those same lines shouldn't adjustedAssetsRepoBranch be changed adjustedAssetsRepoTag?